components/org.wso2.carbon.consent.mgt.endpoint/src/gen/java/org/wso2/carbon/consent/mgt/endpoint/dto/ConsentResponseDTO.java

```diff
@@ -28,6 +28,12 @@
   
   
   private String state = null;
+  
+  
```

Comment:
remove unwanted new lines.

Reply:
These are auto generated codes; swagger to cxf